### PR TITLE
Update google-account-picker.md

### PR DESCRIPTION
**Pull request by Shreyash193**
## Types of changes:
- [x] Correct a spelling
- [x] Add new information
- [x] Change information

## Checklist:
- [x] I have proofread the changes. (Spellings, grammar, etc)
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs/components/google/google-account-picker.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Google Account Picker
+
+## Overview
+A Google Account Picker is a non-visible component which can be used to pick a Google (Gmail) account registered in the user's device. This component can be used to create a login/register like system in the app.
+<br><br>
+![Google Account Picker](/assets/images/components/google/google-account-picker/preview.gif){: class="preview-img" }
+
+
+## Properties
+This component do not have any designer property.
```
Comment (conorshipp) [inline docs/components/google/google-account-picker.md]:
```suggestion
This component doesn't have any properties.
```

File: docs/components/google/google-account-picker.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+## Methods
+
+
+### Pick
+
+![](/assets/images/components//google/google-account-picker/m_pick.png)
```
Comment (conorshipp) [inline docs/components/google/google-account-picker.md]:
```suggestion
![](/assets/images/components/google/google-account-picker/m_pick.png)
```

File: docs/components/google/google-account-picker.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+_**\(**None**\)**_
+
+Picks a Google (Gmail) account registered in the device.
+
+
+## Events
+
+### Picked
+
+![](/assets/images/components//google/google-account-picker/e_picked.png)
```
Comment (conorshipp) [inline docs/components/google/google-account-picker.md]:
```suggestion
![](/assets/images/components/google/google-account-picker/e_picked.png)
```

File: docs/components/google/google-account-picker.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Google Account Picker
+
+## Overview
+A Google Account Picker is a non-visible component which can be used to pick a Google (Gmail) account registered in the user's device. This component can be used to create a login/register like system in the app.
+<br><br>
+![Google Account Picker](/assets/images/components/google/google-account-picker/preview.gif){: class="preview-img" }
```
Comment (conorshipp) [inline docs/components/google/google-account-picker.md]: The images you've linked don't exist in the repo so you'll need to upload them.

Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: I'm new to this. Can you please tell me how to upload images.

Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: I'm new to this. Can you please tell me how to upload images.

Comment (conorshipp) [reply docs/components/google/google-account-picker.md]: This should help you:
https://help.github.com/articles/adding-a-file-to-a-repository/

Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: I'm unable to upload files. It is showing this error:
screenshot_2018-11-15-21-43-10-092_com android chrome

Comment (Sander0542) [reply docs/components/google/google-account-picker.md]: You need to upload the images in the branch you want to push. In this case Shreyash193:patch-1

https://github.com/Shreyash193/Documentation/tree/patch-1/docs/components/google

Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: Done. ✓

File: docs/components/google/google-account-picker.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Google Account Picker
+
+## Overview
+Google Account Picker is a non-visible component which can be used to pick a Google (Gmail) account registered in the user's device. This component can be used to create a login/register like system in the app.
+<br><br>
+![Google Account Picker](/docs/assets/images/google/preview.gif)
```
Comment (barreeeiroo) [inline docs/components/google/google-account-picker.md]: Path to image is wrong. When deployed, the /docs is removed, so consequently the image won't be shown
Remove all references to `/docs` from it

```suggestion
![Google Account Picker](/docs/assets/images/google/preview.gif)
```

File: docs/components/google/google-account-picker.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+This component doesn't have any designer property.
+
+
+
+## Methods
+
+
+### Pick
+
+![Picked](/docs/assets/images/google/m_pick.jpg)
```
Comment (barreeeiroo) [inline docs/components/google/google-account-picker.md]: All images are wrong placed. The path should be `/assets/images/google/google-account-picker/m_pick.jpg`

```suggestion
![Picked](/assets/images/google/google-account-picker/m_pick.jpg)
```

File: docs/components/google/google-account-picker.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+_**\(**None**\)**_
+
+Picks a Google (Gmail) account registered in the device.
+
+
+## Events
+
+### Picked
+
+![Pick](/docs/assets/images/google/e_picked.jpg)
```
Comment (barreeeiroo) [inline docs/components/google/google-account-picker.md]: Suggestion: upload all images as PNG, to keep all images with the same format

File: docs/components/google/google-account-picker.md (hunk 6 above)
Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: But if I don't add "/docs" to it, it is not showing the images.

Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: BTW, I have changed the path to `/assets/images/google/google-account-picker` as you stated above.

File: docs/components/google/google-account-picker.md (hunk 5 above)
Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: Done, changed the path to `/assets/images/google/google-account-picker`.

Comment (Shreyash193) [reply docs/components/google/google-account-picker.md]: @barreeeiroo ...

**Pull request closed without merging**